cpg-language-python/src/test/kotlin/de/fraunhofer/aisec/cpg/frontends/python/PythonFrontendTest.kt

```diff
@@ -60,44 +64,43 @@ class PythonFrontendTest : BaseTest() {
             ) {
                 it.registerLanguage<PythonLanguage>()
             }
-
         assertNotNull(tu)
 
         val p =
             tu.getDeclarationsByName("literal", NamespaceDeclaration::class.java).iterator().next()
         assertNotNull(p)
-        assertEquals("literal", p.name)
+        assertLocalName("literal", p)
 
         val b = p.getDeclarationsByName("b", VariableDeclaration::class.java).iterator().next()
         assertNotNull(b)
-        assertEquals("b", b.name)
+        assertLocalName("b", b)
         assertEquals(TypeParser.createFrom("bool", PythonLanguage()), b.type)
 
         val i = p.getDeclarationsByName("i", VariableDeclaration::class.java).iterator().next()
         assertNotNull(i)
-        assertEquals("i", i.name)
+        assertLocalName("i", i)
         assertEquals(TypeParser.createFrom("int", PythonLanguage()), i.type)
 
         val f = p.getDeclarationsByName("f", VariableDeclaration::class.java).iterator().next()
         assertNotNull(f)
-        assertEquals("f", f.name)
+        assertLocalName("f", f)
         assertEquals(TypeParser.createFrom("float", PythonLanguage()), f.type)
 
         /*
```

Comment:
Why is this commented out?

Reply:
Not sure, but this was already commented out. Probably not handled (yet) -> @maximiliankaul ?

Reply:
Correct. I'll open another PR to implement complex number support for Python.